.github/workflows/docs/intro.txt

```diff
@@ -1,10 +0,0 @@
-pytket-extensions
```

Comment:
What happened to this file?

Reply:
This was unused so I deleted it. I think its a legacy file from when all of the extensions were in the same repo.